### PR TITLE
B2v

### PR DESCRIPTION
**Pull request by jburel**
Changes to work with bump2version
The downside is that the tag will now become v0.5.0 for example
cc @sbesson @will-moore

### CODE REVIEW

**Comment by will-moore** [comment]
I can't think that this would cause any major issues for our users, and if it aligns this app with our other tools it makes everyones lives easier. 👍

File: omero_mapr/utils/version.py [hunk 1]
```diff
@@ -30,12 +30,7 @@ def get_version(version=None):
     See https://www.python.org/dev/peps/pep-0440/
     """
 
-    version = get_full_version(version)
-    parts = 2 if version[2] == 0 else 3
-    res = '.'.join(str(x) for x in version[:parts])
-    if len(version) > 3:
-        res = "%s%s" % (res, version[3])
-    return str(res)
+    return get_full_version(version)
```
Comment (sbesson) [inline omero_mapr/utils/version.py:33]: The only other place where this API is being used is `custom_context_processor` but I don't know more about the context so I cannot comment on whether these changes would affect the usage

Comment (jburel) [reply omero_mapr/utils/version.py:33]:
```
def mapr_url_suffix(request):
    suffix = u"?_%s" % get_version()
    return {'mapr_url_suffix': suffix}
```

Comment (jburel) [reply omero_mapr/utils/version.py:33]: I don't think it will be anything

Comment (sbesson) [reply omero_mapr/utils/version.py:33]: :+1: happy to follow your lead here.

**Review by sbesson — APPROVED** [review]

**Pull request merged**